modules/templates/helper.go

```diff
@@ -550,12 +553,25 @@ func AvatarHTML(src string, size int, class, name string) template.HTML {
 
 // SVG render icons - arguments icon name (string), size (int), class (string)
 func SVG(icon string, others ...interface{}) template.HTML {
-	size, class := parseOthers(16, "", others...)
+	sizeInt, class := parseOthers(16, "", others...)
+
+	size := "16"
+
+	// if sizeInt returned the default(16), it's possible that the first argument
+	// is a string, and thus has a size with a unit.
+	if sizeInt == 16 && len(others) > 0 {
```

Comment:
I chose to do the "string" checking here, instead of `parseOthers` as the other 3 references of that function really requires that the output is a int and not a int with a unit.